### PR TITLE
Update IID option

### PR DESCRIPTION
**Pull request by lynnfel**
Here are the changes we discussed to the IID option.  You will see one "TO DO" note from me.   I was surprised by the sentence about the Image Display (after receiving RAD-106) retrieving images from the Image Manager.  I see a dotted line also added to the actor transaction diagram.  This is new, I think. (and I don't like the unlabelled dotted line between the ID and IM)

### CODE REVIEW

File: input/pagecontent/volume-1.md [hunk 1]
```diff
@@ -186,19 +186,21 @@ The requests shall use the information in DiagnosticReport.result.derivedFrom.en
...
 
-A Report Reader that supports this option shall use Invoke Image Display [RAD-106] to invoke an Image Display which in turn retrieves images from an Image Manager / Image Archive. Report Readers shall translate the embedded image references and endpoints in the DiagnosticReport resource into corresponding Invoke Image Display URLs.
+A Report Reader that supports this option is grouped with an IID Image Display Invoker and shall use the Invoke Image Display [RAD-106] transaction to invoke an external Image Display.  The Report Reader shall translate the embedded image references and endpoints in the DiagnosticReport resource into corresponding Invoke Image Display [RAD-106] URLs.
 
-> Note: For this option to work, the Image Display also needs to support the [IID Invoke Option](#xx27-iid-invoke-option).
+An Image Display that supports this option shall support the Invoke Image Display [RAD-106] transaction as an Image Display.  <<TO DO:  IS THIS TRUE?>>  When the Image Display receives a [RAD-106] request, it retrieves the requested images from an Image Manager / Image Archive.
```
Comment (kinsonho) [inline input/pagecontent/volume-1.md:195]: I looked at the display requirements in RAD-106. I think it all applies to IMR Image Display. As discussed, I will add a new IHE_IMR viewer type in RAD-106 as well.

**Comment by kinsonho** [comment]
> Here are the changes we discussed to the IID option. You will see one "TO DO" note from me. I was surprised by the sentence about the Image Display (after receiving RAD-106) retrieving images from the Image Manager. I see a dotted line also added to the actor transaction diagram. This is new, I think. (and I don't like the unlabelled dotted line between the ID and IM)

@lynnfel , IID does not indicate how the Image Display get the images. So I can okay to remove the last sentence and the dotted line if that is clear. The dotted line is there exactly because IID does not indicate how does the Image Display get the images.

File: input/pagecontent/volume-1.md (hunk 1 above)
Comment (kinsonho) [reply input/pagecontent/volume-1.md:195]: All other changes look good.

**Comment by lynnfel** [comment]
In IMR as written today, the IMR on the receiving end of RAD-106 is an IMR Image Display.  It is not an IID Image Display, nor is it grouped with an IID Image Display.  This is a subtle difference.

For the Image Display receiving RAD-106, do you think this statement is true?  "Note 2: The Image Display shall support at least one of WADO-RS Retrieve [RAD-107] or Retrieve Images [RAD-16]". (from Note 2 under the transaction table)   

OR, is it true that IMR does not specify how that Image DIsplay gets access to its images?

**Pull request merged**